components/utils.py

```diff
@@ -1223,3 +1223,79 @@ def verify_service_http(service_name, url, predicate=None):
         return check_http_response(url, predicate)
     except (IOError, ValueError) as e:
         ctx.abort_operation('{0} error: {1}: {2}'.format(service_name, url, e))
+
```

Comment:
are these functions used anywhere else in the blueprint? If not, why not move them to a different file and import it for each creaction_validation script?

Also, the name creaction_validation is misleading.. as it doesn't validate the creation of anything